backend/handlers/handlers.go

```diff
@@ -829,8 +829,12 @@ func (handlers *Handlers) getAccountSummary(_ *http.Request) (interface{}, error
 	fiat := handlers.backend.Config().AppConfig().Backend.MainFiat
 	// Chart data until this point in time.
 	until := handlers.backend.RatesUpdater().HistoryLatestTimestampAll(handlers.allCoinCodes(), fiat)
-	if until.IsZero() {
+	if until.IsZero() || time.Since(until) > 2*time.Hour {
```

Comment:
Just an observation: as a user, I'd definitely prefer *some* data instead of none. But maybe it's just me.

Reply:
Yeah me too, but if the data is too old, adding the `now`-datapoint is bad. Not sure how to solve this.

Maybe after we get rid of cryptocompare, we can revert this.